datasets/esa-worldcover.yaml

```diff
@@ -1,10 +1,10 @@
 Name: ESA WorldCover 
-Description: The European Space Agency (ESA) WorldCover is a global land cover map with 11 different land cover classes produced at 10m resolution based on combination of both Sentinel-1 and Sentinel-2 data. In areas where Sentinel-2 images are covered by clouds for an extended period of time, Sentinel-1 data then provides complimentary information on the structural characteristics of the observed land cover. Therefore, the combination of Sentinel-1 and Sentinel-2 data makes it possible to update the land cover map almost in real time. WorldCover Map has been produced for 2020 (01 January to 31 December) with a global coverage as part of the 5th Earth Observation Envelope Programme (EOEP-5). It provides valuable information for applications such as biodiversity, food security, carbon assessment and climate modelling. More information can be found on the [WorldCover website](https://esa-worldcover.org/en) and the product [User Manual](https://esa-worldcover.s3.amazonaws.com/v100/2020/docs/WorldCover_PUM_V1.0.pdf).  
+Description: The European Space Agency (ESA) WorldCover product provides global land cover maps for 2020 & 2021 at 10 m resolution based on Copernicus Sentinel-1 and Sentinel-2 data. The WorldCover product comes with 11 land cover classes and has been generated in the framework of the ESA WorldCover project, part of the 5th Earth Observation Envelope Programme (EOEP-5) of the European Space Agency. A first version of the product (v100), containing the 2020 map was released in October 2021. The 2021 map was released in October 2022 using an improved algorithm (v200). The WorldCover 2020 and 2021 maps were generated with different algorithm versions and therefore changes between the maps should be treated with caution as these contain both real land cover changes as well as changes due to the used algorithms.
 
-Documentation: Documentation is available [here](https://esa-worldcover.s3.amazonaws.com/readme.html).
+Documentation: Documentation is available [here](https://esa-worldcover.org/en/data-access).
 Contact: https://esa-worldcover.org/en/contact
-ManagedBy: "[Sinergise on behalf of VITO](https://vito.be)"
-UpdateFrequency: Not yet defined.
+ManagedBy: "[VITO](https://vito.be)"
+UpdateFrequency: Yerly.
 Collabs:
   ASDI:
```

Comment:
@apprivet do you approve? It's the same as https://registry.opendata.aws/esa-worldcover/ plus 2021.